### ISSUE TITLE
Doesn't work with instagram apk

### ISSUE THREAD

**Issue by PoojanThaker** [hojatsajadinia/AndRoPass#11]
After I run it on instagram apk, it compiles but I'm not able to install using the modified apk now.

Error on blue stacks:
![image](https://github.com/user-attachments/assets/d74e4b05-a30a-42d8-a116-11f974ccbc55)

Docker stdout:
[INFO] Checking AndRoPass requirements
[INFO] Java binary found: "17.0.13" 2024-10-15\nOpenJDK Runtime Environment (build 17.0.13+11-alpine-r0)\nOpenJDK 64-Bit Server VM (build 17.0.13+11-alpine-r0, mixed mode, sharing)\n'
[INFO] APKTool binary found
[INFO] Uber APK Signer binary found
[INFO] Decompiling application
[INFO] 23 Root Detection Pattern(s) Detected.
[INFO] 49 Emulator Detection Pattern(s) Detected.
[ERROR] Unable to scan the application
[INFO] Compiling application
[INFO] Signing application
[DONE] Application Out Path: /data/AndRoPass_WOR_instagram-aligned-debugSigned.apk

Link to Original and Converted APKs : https://file.io/3zY6Gi7rm7AV

**Comment by PoojanThaker** [comment]
But I'm still able to install the original .apk (its downloaded from apkmirror).

**Comment by mzalazar** [comment]
Yes, but the modified one hasn't any signature. Use uber signer because it seems that AndRoPass doesn't do a good job signing apk.
